examples/sample_run_local_tools_groq.py

```diff
@@ -0,0 +1,60 @@
+"""Groq Sample"""
+import os
+from typing import List
+from dotenv import load_dotenv
+from groq import Groq
+from toolhouse import Toolhouse
+load_dotenv()
+
+TOKEN = os.getenv("GROQCLOUD_API_KEY")
+TH_TOKEN = os.getenv("TOOLHOUSE_BEARER_TOKEN")
+
+
+local_tools = [
+    {'type': 'function',
+     'function':
+         {
+             'name': 'hello',
+             'description': 'The user receive a customized hello message from a city and return it to the user',
```

Comment:
Grammar errors. Should be: “The user receives a customized hello message from a city and returns it to the user.”

Reply:
Fiexd